addons/test_event_full/tests/test_performance.py

```diff
@@ -280,7 +280,7 @@ def test_registration_create_batch_website(self):
         form like) """
         event = self.env['event.event'].browse(self.test_event.ids)
 
-        with freeze_time(self.reference_now), self.assertQueryCount(event_user=651):  # tef 647 - com 649
+        with freeze_time(self.reference_now), self.assertQueryCount(event_user=693):  # tef 689 - com 691
```

Comment:
:grimacing:

Reply:
Good news, in the new version, test_event_full is not impacted anymore.